api/groups/eventsGroup_test.go

```diff
@@ -27,8 +27,8 @@ const eventsPath = "/events"
 
 func createMockEventsGroupArgs() groups.ArgsEventsGroup {
 	return groups.ArgsEventsGroup{
-		Facade:            &mocks.FacadeStub{},
-		EventsDataHandler: &mocks.EventsDataHandlerStub{},
+		Facade:         &mocks.FacadeStub{},
+		PayloadHandler: &testscommon.PayloadHandlerStub{},
```

Comment:
Please do not import stubs from other projects.
I know it is cumbersome to re-write them again in your project, but it's for the better. This way, if there is something changed in the `communication` satellite project, your tests will still work